build-reporter-plugin.js

```diff
@@ -53,21 +54,25 @@ class BugsnagBuildReporterPlugin {
   getBuildOpts (opts) {
     // Required options
     const buildOpts = {
-      apiKey: opts.build.apiKey,
-      versionName: opts.build.appVersion
+      apiKey: opts.build.apiKey || opts.options.apiKey,
+      versionName: opts.build.appVersion || opts.options.appVersion
```

Comment:
These were never an option in the second parameter, so don't need to be added. it's only when they were in `opts.options` that we need to add them to `opts.build`

```suggestion
      apiKey: opts.build.apiKey,
      versionName: opts.build.appVersion
```